modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/model/Translation.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.translation.translator.deepl.internal.model;
+
+/**
+ * @author Yasuyuki Takeo
+ */
+public class Translation {
+
+	// e.g. EN
+
+	public String detected_source_language;
+	public String text;
```

Comment:
Remove comments; make this a proper pojo, etc.

Reply:
@adolfopa same as [this](https://github.com/liferay-lima/liferay-portal/pull/3422#discussion_r962579320)